tests/dialects/test_databricks.py

```diff
@@ -28,6 +28,13 @@ def test_databricks(self):
             "SELECT * FROM sales UNPIVOT EXCLUDE NULLS (sales FOR quarter IN (q1 AS `Jan-Mar`))"
         )
 
+        self.validate_all(
+            "CREATE FUNCTION greet(name STRING) RETURNS STRING LANGUAGE PYTHON AS $$def greet(name):\n\t return f'Hello {name}!'$$",
+            write={
+                "snowflake": "CREATE FUNCTION greet(name TEXT) RETURNS TEXT LANGUAGE PYTHON AS 'def greet(name):\n\t return f\\'Hello {name}!\\''"
+            },
+        )
+
```

Comment:
This test is no good, we're not testing the roundtrip from Databricks to Databricks. Have you double-checked that this is valid Snowflake?

Reply:
you should also test the here docs version